resources/helm/v2.4/istio-telemetry/prometheus/templates/deployment.yaml

```diff
@@ -128,7 +127,42 @@ spec:
           - mountPath: /etc/istio-certs
             name: istio-certs
           {{- end }}
-
+        - name: config-reloader
```

Comment:
Prometheus operator deploys `config-reloader` as an init container. Can it be used as a standard container?